jerry-main/main-unix.c

```diff
@@ -276,6 +286,14 @@ main (int argc,
     {
       jerry_port_default_set_abort_on_fail (true);
     }
+    else if (!strcmp ("--no-prompt", argv[i]))
+    {
+      no_prompt = true;
+    }
+    else if (!strcmp ("-", argv[i]))
```

Comment:
Without this `if`, execution does not reach the final `else` branch because the "error handling if" (just below) catches everything that starts with a dash and has not been recognized hitherto.